src/main/java/io/xstefank/wildfly/bot/util/DescriptionStringBuilder.java

```diff
@@ -0,0 +1,23 @@
+package io.xstefank.wildfly.bot.util;
+
+public class DescriptionStringBuilder {
+
+    StringBuilder sb;
+    public DescriptionStringBuilder() {
+        sb = new StringBuilder();
+    }
+
+    public DescriptionStringBuilder append(String message) {
+        sb.append(message);
+        return this;
+    }
+
+    public DescriptionStringBuilder appendQuote(String message) {
```

Comment:
this is not a Quote and I still don't understand why we need a separate class for this override

Reply:
I will rename it to `appendBlockQuote`, as that is what it is called in markdown. which we are using it's syntax for writing comments / editing descriptions. Also it might be easier for future expansion. Currently it is also used in tests and makes the, more readable.